decidim/decidim
# Add manifest name check for spaces and components

#### :tophat: What? Why?
During upgrade to decidim 0.29, we have noticed there are some exceptions due to fact that Spaces and Components have been removed. This PR aims to fix those exceptions by adding a scope to component so that we can identify only the records that belong to register manifests.
This will also fix the issue reported [here](https://github.com/decidim/decidim/pull/13236#issuecomment-2257653908):

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #12366
- Related to #11171

#### Testing
1. Create a development application 
2. Patch the generator file with something like the example below, which will simulate that the accountability & initiatives modules have been uninstalled / removed 
3. Start the dev server & see the errors 
4. Apply the patch
5. Run the tasks 
6. See there is no error 

#### Example
```
# expanded version of `gem "decidim", path: ".."`
# gem "decidim-accountability", path: ".."
gem "decidim-admin", path: ".."
gem "decidim-api", path: ".."
gem "decidim-assemblies", path: ".."
gem "decidim-blogs", path: ".."
gem "decidim-budgets", path: ".."
gem "decidim-comments", path: ".."
gem "decidim-core", path: ".."
gem "decidim-debates", path: ".."
gem "decidim-forms", path: ".."
gem "decidim-generators", path: ".."
gem "decidim-meetings", path: ".."
gem "decidim-pages", path: ".."
gem "decidim-participatory_processes", path: ".."
gem "decidim-proposals", path: ".."
gem "decidim-sortitions", path: ".."
gem "decidim-surveys", path: ".."
gem "decidim-system", path: ".."
gem "decidim-verifications", path: ".."
# eof expanded version of `gem "decidim", path: ".."`

gem "decidim-conferences", path: ".."
gem "decidim-design", path: ".."
# gem "decidim-initiatives", path: ".."
gem "decidim-templates", path: ".."
```

:hearts: Thank you!
